ui/utils/reportError.ts

```diff
@@ -0,0 +1,26 @@
+export default function reportError(error: Error) {
+  const { message, stack } = error;
+  const hookUrl = process.env.ERROR_REPORTING_WEBHOOK;
+  if (hookUrl) {
+    const content = JSON.stringify({
+      error,
+      message,
+      stack,
+      url: window?.location,
+    });
```

Comment:
If we do `JSON.stringify(..., nulll, 2)` then it'll be easier to read on discord